kdbg/sourcewnd.cpp

```diff
@@ -777,6 +785,24 @@ void SourceWindow::setTabWidth(int numChars)
     setTabStopWidth(w);
 }
 
+void SourceWindow::storeLineToVec(int line)
+{
+    m_expandedLines.push_back(line);
+}
+
+void SourceWindow::deleteLineFromVec(int line)
+{
+    m_expandedLines.erase(std::remove(m_expandedLines.begin(), m_expandedLines.end(),line), m_expandedLines.end());
+}
+
+void SourceWindow::restorePrevDisass()
+{
+    for(auto& lineNo : m_expandedLines)
+    {
+        QTimer::singleShot(500, [this, lineNo](){actionExpandRow(lineNo);});
+    }
+}
+
```

Comment:
Nice!

Reply:
Thank you, I hope a vector is a good solution and not very expensive. I couldn't imagine a better way to "remember" what lines were expanded before reloading the view.

Reply:
I think a vector strikes the balance quite well.